dependency/health_service.go

```diff
@@ -261,6 +271,9 @@ func (d *HealthServiceQuery) String() string {
 	if d.peer != "" {
 		name = name + "@peer=" + d.peer
 	}
+	if d.samenessGroup != "" {
+		name = name + "@samenessGroup=" + d.samenessGroup
```

Comment:
nit: do these usually use camel case or dash-separated?

Reply:
fixed in a2260c48e2712a81c5e0846d0527428ed257557c